envs/example/ci-full/group_vars/all.yml

```diff
@@ -16,13 +29,46 @@ neutron:
   lbaas:
     enabled: True
 
+common:
+  hwraid:
+    enabled: False
+
+logging:
+  enabled: true
```

Comment:
only need this variable. Rest all in this section is role defaults so not need to define it here.